internal/xds/rbac/converter.go

```diff
@@ -78,6 +81,9 @@ func convertCustomConfig(typeURL string, s *structpb.Struct) (json.RawMessage, s
 	// type_url field in the TypedStruct. We get this by using the part after
 	// the last / character. Can assume a valid type_url from the control plane.
 	urls := strings.Split(typeURL, "/")
+	if len(urls) == 0 {
+		return nil, "", fmt.Errorf("error converting custom lb policy %v for %v: typeURL must have a url-like format with the typeName being the value after the last /", typeURL, s)
+	}
 	name := urls[len(urls)-1]
 
 	rawJSON, err := json.Marshal(s)
```

Comment:
What happens if `s == nil`? In [C++](https://github.com/grpc/grpc/blob/6818c8740ff49432b1b582ea66adb1006a4672c1/src/core/ext/xds/xds_common_types.cc#L491), it defaults to an empty json object. We need to ensure the same behavior.

Reply:
Done, feels a little messy to have to pre-declare, but that gets around a needless json.Marshal call.
PTAL